Updated secondary slot algo doc (#6297)
w3f/polkadot-wiki: docs/learn/learn-consensus.md

### No Validators in Slot

When no validators have rolled low enough in the randomness lottery to qualify for block production,

> a slot can remain seemingly blockless. We run a secondary validator selection algorithm in the 
> background to prevent this. The validators selected through this predictable algorithm always produce 
> blocks. Still, these _secondary_ blocks are ignored if the same slot

Reviewer: ```suggestion
a slot can remain seemingly blockless. Polkadot runs a [secondary validator selection algorithm](https://spec.polkadot.network/sect-block-production#defn-babe-secondary-slots) in the 
background. The validators selected through this predictable algorithm always produce 
blocks. These _secondary_ blocks are ignored if the same slot
```
